scipy/linalg/decomp_qr.py

```diff
@@ -84,28 +84,28 @@ def qr(a, overwrite_a=False, lwork=None, mode='full', pivoting=False,
 
     Examples
     --------
-    >>> from scipy import random, linalg, dot, diag, all, allclose
+    >>> from scipy import linalg
     >>> a = random.randn(9, 6)
```

Comment:
```suggestion
    >>> a = np.random.randn(9, 6)
```

From [Travis](https://travis-ci.org/scipy/scipy/jobs/560502178#L1372):
```
File "build/testenv/lib/python3.6/site-packages/scipy/linalg/decomp_qr.py", line 88, in qr
Failed example:
    a = random.randn(9, 6)
```